app/views/sign_up/registrations/_required_pii_accordion.html.erb

```diff
@@ -40,7 +40,7 @@
   <p class="grid-row grid-gutter">
     <%= image_tag(
           asset_url('get-started/personal-details.svg'),
-          size: '40', alt: '', class: 'margin-top-05 grid-col-auto',
+          size: '40', alt: t('devise.registrations.start.bullet_3_img'), class: 'margin-top-05 grid-col-auto',
```

Comment:
I _think_ it's correct as-is, but just to double-check, this is meant to be 3, and not 4? Since the sequence here is `bullet_3_img`, `bullet_3_img`, `bullet_5_img`. I guess since it's the same image as the previous bullet point, it's sensible for it to also have the same alt text?

Reply:
yeah I was on the fence about whether to change the name to something like "bullet_3_4_img" to make it clearer but ultimately thought that was still a little weird and it would be redundant to have it written in the translation doc twice

Reply:
Yeah and as mentioned in the previous pull request, it'd be good to move away from the "bullet_#" convention anyways, so wouldn't want to make that harder.

All good then 👍